packet_queue/nfqueue.py

```diff
@@ -48,31 +52,33 @@ def configure(protocol, port, pipes, interface):
       raise ValueError("Given interface does not exist.", interface)
 
   add(protocol, port, interface)
-  queue = netfilterqueue.NetfilterQueue()
+  manager = libnetfilter_queue.Manager()
```

Comment:
I'm not sure I understand what the point of this instantiation is -- we aren't returning this instance, and it's not being used directly to set self.manager. (What did queue = ... originally do?)